src/licensedcode/data/licenses/opl-uk-3.0.LICENSE

```diff
@@ -0,0 +1,125 @@
+---
+key: opl-uk-3.0
+short_name: United Kingdom Open Parliament Licence v3.0
+name: United Kingdom Open Parliament Licence v3.0
+category: Permissive
+owner: Parliament of United Kingdom
+spdx_license_key: OPL-UK-3.0
+other_urls:
+    - https://www.parliament.uk/site-information/copyright-parliament/open-parliament-licence/
+---
+
+United Kingdom Open Parliament Licence v3.0
```

Comment:
This does not look like a software license? :thinking:

Reply:
@AyanSinhaMahapatra    The entry at https://spdx.org/licenses/OPL-UK-3.0.html   includes the full text and it is ok to consider it a software license.